### DISCUSSION TITLE
Citing Issue

### DISCUSSION THREAD

**Discussion by WaseemSSaeed** [Qiskit/.github#28] [General Feedback]
Hello :)
Could you please help me in how to cite QisKit textbook correctly :) for example, I cannot find the year. Please if you could provide me with a full suitable APA citing of QisKit textbook for a given section. or any citing system, just to see how you do it.
Kind regards

**Comment by 1ucian0** [comment]
Hi Wassen! This is a known issue https://github.com/qiskit-community/qiskit-textbook/issues/1339 (I will add more info so the community can start working on this soon).

About your specific year request, the time the textbook got online (maybe we can call it "first edition") was in [Dec 2019](https://web.archive.org/web/20191220042259/https://qiskit.org/textbook/preface.html). In general, I think is usual to set the year to the visited date when citing websites. For example, I generated this bib ref with [scribbr](https://www.scribbr.com/apa-examples/website/):

```
@misc{textbook-2022,
	author = {given-i=T.Q., given={Qiskit}, family=Team},
	month = {10},
	title = {{Phase Kickback}},
	year = {2022},
	url = {https://qiskit.org/textbook/ch-gates/phase-kickback.html},
}
```
